### PR TITLE
Move from leap to tumbleweed

### PR DESCRIPTION
**Pull request by Itxaka**
Fixes (When it works): https://github.com/rancher-sandbox/cOS-toolkit/issues/1257

Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

File: packages/firmware/collection.yaml [hunk 1]
```diff
@@ -10,12 +10,12 @@ packages:
       autobump.string_replace: '{ "prefix": "" }'
       autobump.prefix: "prefix"
       autobump.hook: |
-        curl -s -L https://download.opensuse.org/distribution/leap/15.3/repo/oss/$(curl -s -L https://download.opensuse.org/distribution/leap/15.3/repo/oss/repodata/repomd.xml | dasel select -p xml 'repomd.data.[0].location.-href') | gunzip | dasel -r xml -p json | jq '.metadata.package[] | select(.name=="u-boot-rpiarm64") | select(.arch!="src").version | map(.)[2] + "-" +  map(.)[1]' -r | tail -n1
```
Comment (mudler) [inline packages/firmware/collection.yaml:13]: nope, not for firmwares. we have been already there, and seems Leap ones get more attention, TW got broken for RPIs several times

Comment (Itxaka) [reply packages/firmware/collection.yaml:13]: really? How weird, I would not expect that! Reverting this!

File: values/teal-x86_64.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ codename: "teal"
 arch: "x86_64"
 golang_arch: "amd64"
 
-tool_image: opensuse/leap:15.3
```
Comment (mudler) [inline values/teal-x86_64.yaml:7]: I'd be tempted to stay stable on teal, and avoid tw or latest tags. We can use the other flavors to assess issues with newer images

**Comment by mudler** [comment]
Needs a bump to the cos collection for a good test :clown_face:

**Review by mudler — APPROVED** [review]
Looks good!

**Comment by Itxaka** [comment]
well, it fails incredibly...

**Comment by Itxaka** [comment]
```
#8 [4/5] RUN zypper in -y --no-recommends grub2-i386-pc grub2-x86_64-efi grub2 curl tar xz git
       #8 sha256:bbbb63be7aaebdea5c8499111de8918f42d25cb2a35afb7be7e9725b7162da39
 INFO   #8 0.409 terminate called after throwing an instance of 'std::system_error'
       #8 0.409   what():  Operation not permitted
```

works locally :thinking:

File: packages/firmware/collection.yaml (hunk 1 above)
Comment (Itxaka) [reply packages/firmware/collection.yaml]: done!

File: values/teal-x86_64.yaml (hunk 2 above)
Comment (Itxaka) [reply values/teal-x86_64.yaml]: done!

**Pull request merged**